GVFS/GVFS.Common/NuGetUpgrader/ReleaseManifest.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Newtonsoft.Json;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace GVFS.Common.NuGetUpgrader
+{
+    /// <summary>
+    /// Details on the upgrade included in this package, including information
+    /// on what packages are included and how to install them.
+    /// </summary>
+    public class ReleaseManifest
+    {
+        public static string WindowsPlatformKey = "Windows";

```

Comment:
I only see this being used in the unit tests, can it be removed from this class?

Reply:
There was a place in product code that used a string literal for this value - I updated that location to use this field (and turned it into a const value).